### ISSUE TITLE
How will the Chrome Manifest V2 deprecation affect Cypress 12 tests?

### ISSUE THREAD

**Issue by alexsch01** [cypress-io/cypress#28405]
### What would you like?

Will commands such as `npx cypress run --browser chrome` still work?

Thank you

**Comment by jennifer-shehane** [comment]
For reference I'll cite Chrome's blog post about the deprecation: https://developer.chrome.com/blog/resuming-the-transition-to-mv3/

We updated our Chrome extension to Manifest V3 in the Cypress App in v13.3.1. With the deprecation of the Manifest V2 extensions coming in June 2024, this mean that the Cypress App will not function with newer Chrome browsers that no longer support Manifest V2 (Chrome version to be announced, but > Chrome v127). Users will be required to update the Cypress App to v13.3.1 or later in order to test in later Chrome browser versions.

**Comment by alexsch01** [comment]
@jennifer-shehane 
I'm a bit confused...

```
on('before:browser:launch', (browser, launchOptions) => {
    launchOptions.args.push('--disable-extensions')
    return launchOptions
});
```

in the config file.....

`cypress run --browser chrome` and [`cypress open` + picking Chrome] still works

**Comment by alexsch01** [comment]
@jennifer-shehane 

Is the Chrome extension that comes with Cypress needed to be able to run tests properly?

**Comment by jennifer-shehane** [comment]
@alexsch01 I'm going to have to verify the behavior actually - looking at our v3 extension, it seems like CDP was already handling a lot of what our old extension was doing, so the way it breaks and when is going to be a bit fuzzier. I plan to have us put some effort into understanding and sharing how this will break people in Q1.

**Comment by alexsch01** [comment]
@jennifer-shehane Thanks for your response

I just did a very basic test on Cypress 10.0.0 running on Chromium 119 with --disable-extensions flag, it worked fine

**Comment by jennifer-shehane** [comment]
Yah our old extension used to handle things like cookies, screenshots, and downloads - so mostly some of those functionalities would not function. But at some point in the past we replaced those functionalities with CDP, although the code was still sitting there in the V2 extension. So we'd have to figure out the version of Chrome where those functionalities didn't function in CDP and when we introduced that replacement to figure out how it might break people's cookies, screenshots, and downloads in what version of Cypress and what version of Chrome.
